src/layouts/DialogLayout/DialogLayout.stories.tsx

```diff
@@ -7,7 +7,7 @@ import * as React from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
 
 import { Dialog } from '../../components/containers/Dialog/Dialog.tsx';
-import { Logo } from '../../layouts/AppLayout/Logo/Logo.tsx';
+import { Logo } from '../../fortanix/Logo/Logo.tsx';
```

Comment:
While we're at it let's update this file so that it uses the logo design from the Figma.